### PR TITLE
Docs: mention mimirtool commands in HTTP API doc

### LINKED ISSUE #2153: Docs: API reference should recommend to use mimirtool for ruler and alertmanager API
Mimir users look at the [HTTP API reference doc](https://grafana.com/docs/mimir/next/operators-guide/reference-http-api/) also as a guide to set ruler and alertmanager config, but manually crafting a `curl` command could be difficult. We could recommend to use `mimirtool` for ruler and alertmanager endpoints.

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does
See #2153

#### Which issue(s) this PR fixes or relates to

Fixes #2153

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/operators-guide/tools/mimirtool.md [hunk 1]
```diff
@@ -199,13 +199,13 @@ To perform a trial run that does not make changes, you can use the dry run flag
...
 
-The format of the file is the same format as shown in [rules load](#load).
+The format of the file is the same format as shown in [rules load](#load-rule-groups).
 
 #### Prepare
 
 This `prepare` command prepares a rules file that you upload to Grafana Mimir.
 It lints all PromQL expressions and adds a label to your PromQL query aggregations in the file.
-The format of the file is the same format as shown in [rules load](#load).
+The format of the file is the same format as shown in [rules load](#load-rule-groups).
```
Comment (pstibrany) [inline docs/sources/operators-guide/tools/mimirtool.md:208]: I don't see where `#load-rule-groups` should point in this document. Also sometimes we use `{{< relref "#..." >}}`, which one is correct?

**Review by pstibrany — APPROVED** [review]

**Review by pstibrany — APPROVED** [review]
LGTM, but see if we should use `{{< relref "#..." >}}` instead.

File: docs/sources/operators-guide/tools/mimirtool.md (hunk 1 above)
Comment (pracucci) [reply docs/sources/operators-guide/tools/mimirtool.md:208]: There's no practical difference when it comes to self-page anchors.

Comment (pstibrany) [reply docs/sources/operators-guide/tools/mimirtool.md:208]: I thought `relref` links get checked, but if there is no difference, I'd suggest to use `[...](#...)` form everywhere, as it doesn't confuse other tools (eg. Goland). But that's not for this PR.

File: docs/sources/operators-guide/reference-http-api/index.md [hunk 2]
```diff
@@ -576,6 +576,8 @@ This endpoint can be disabled via the `-ruler.enable-api` CLI flag (or its respe
 
 Requires [authentication](#authentication).
 
+> **Note:** We recommend that you use [`mimirtool rules list` command]({{< relref "../tools/mimirtool.md#list-rules" >}}) to list all rule groups from Mimir.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/reference-http-api/index.md:579]:
```suggestion
> **Note:** To list all rule groups from Mimir, use [`mimirtool rules list` command]({{< relref "../tools/mimirtool.md#list-rules" >}}).
```

File: docs/sources/operators-guide/reference-http-api/index.md [hunk 3]
```diff
@@ -669,6 +671,8 @@ This endpoint can be disabled via the `-ruler.enable-api` CLI flag (or its respe
 
 Requires [authentication](#authentication).
 
+> **Note:** We recommend that you use [`mimirtool rules get` command]({{< relref "../tools/mimirtool.md#get-rule-group" >}}) to retrieve a single rule group from Mimir.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/reference-http-api/index.md:674]:
```suggestion
> **Note:** To retrieve a single rule group from Mimir, use [`mimirtool rules get` command]({{< relref "../tools/mimirtool.md#get-rule-group" >}}) .
```

File: docs/sources/operators-guide/reference-http-api/index.md [hunk 4]
```diff
@@ -683,6 +687,8 @@ This endpoint can be disabled via the `-ruler.enable-api` CLI flag (or its respe
 
 Requires [authentication](#authentication).
 
+> **Note:** We recommend that you use [`mimirtool rules load` command]({{< relref "../tools/mimirtool.md#load-rule-groups" >}}) to load one or more rule groups into Mimir.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/reference-http-api/index.md:690]:
```suggestion
> **Note:** To load one or more rule groups into Mimir, use [`mimirtool rules load` command]({{< relref "../tools/mimirtool.md#load-rule-groups" >}}) .
```

File: docs/sources/operators-guide/reference-http-api/index.md [hunk 5]
```diff
@@ -709,6 +715,8 @@ This endpoint can be disabled via the `-ruler.enable-api` CLI flag (or its respe
 
 Requires [authentication](#authentication).
 
+> **Note:** We recommend that you use [`mimirtool rules delete` command]({{< relref "../tools/mimirtool.md#delete-rule-group" >}}) to delete a rule group from Mimir.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/reference-http-api/index.md:718]:
```suggestion
> **Note:** To delete a rule group from Mimir use [`mimirtool rules delete` command]({{< relref "../tools/mimirtool.md#delete-rule-group" >}}).
```

File: docs/sources/operators-guide/reference-http-api/index.md [hunk 6]
```diff
@@ -795,6 +803,8 @@ This endpoint can disabled enabled via the `-alertmanager.enable-api` CLI flag (
 
 Requires [authentication](#authentication).
 
+> **Note:** We recommend that you use [`mimirtool alertmanager get` command]({{< relref "../tools/mimirtool.md#get-alertmanager-configuration" >}}) to retrieve a tenant's Alertmanager configuration from Mimir.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/reference-http-api/index.md:806]:
```suggestion
> **Note:** To retrieve a tenant's Alertmanager configuration from Mimir, use [`mimirtool alertmanager get` command]({{< relref "../tools/mimirtool.md#get-alertmanager-configuration" >}}).
```

File: docs/sources/operators-guide/reference-http-api/index.md [hunk 7]
```diff
@@ -809,6 +819,8 @@ This endpoint can disabled enabled via the `-alertmanager.enable-api` CLI flag (
 
 Requires [authentication](#authentication).
 
+> **Note:** We recommend that you use [`mimirtool alertmanager load` command]({{< relref "../tools/mimirtool.md#load-alertmanager-configuration" >}}) to load a tenant's Alertmanager configuration to Mimir.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/reference-http-api/index.md:822]:
```suggestion
> **Note:** To load a tenant's Alertmanager configuration to Mimir, use [`mimirtool alertmanager load` command]({{< relref "../tools/mimirtool.md#load-alertmanager-configuration" >}}).
```

File: docs/sources/operators-guide/reference-http-api/index.md [hunk 8]
```diff
@@ -847,6 +859,8 @@ This endpoint can be disabled via the `-alertmanager.enable-api` CLI flag (or it
 
 Requires [authentication](#authentication).
 
+> **Note:** We recommend that you use [`mimirtool alertmanager delete` command]({{< relref "../tools/mimirtool.md#delete-alertmanager-configuration" >}}) to delete a tenant's Alertmanager configuration from Mimir.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/reference-http-api/index.md:862]:
```suggestion
> **Note:** To delete a tenant's Alertmanager configuration from Mimir, use [`mimirtool alertmanager delete` command]({{< relref "../tools/mimirtool.md#delete-alertmanager-configuration" >}}).
```

File: docs/sources/operators-guide/tools/mimirtool.md [hunk 9]
```diff
@@ -127,39 +127,39 @@ The commands in this section enable you to perform the following actions:
...
+#### Delete rule group
 
 The following command deletes a rule group.
 
 ```bash
 mimirtool rules delete <namespace> <rule_group_name>
 ```
 
-#### Load
+#### Load rule groups
```
Comment (osg-grafana) [inline docs/sources/operators-guide/tools/mimirtool.md:162]: Should this be singular or plural, to provide parallel structure wrt language in subheadings?

File: docs/sources/operators-guide/tools/mimirtool.md (hunk 1 above)
Comment (osg-grafana) [reply docs/sources/operators-guide/tools/mimirtool.md:208]: (Internal) See https://github.com/grafana/technical-documentation/issues/404.

Comment (osg-grafana) [reply docs/sources/operators-guide/tools/mimirtool.md:208]: @gguillotte-grafana, there might or might not be link checking for `ref`, `relref`, or both.

File: docs/sources/operators-guide/tools/mimirtool.md (hunk 9 above)
Comment (osg-grafana) [reply docs/sources/operators-guide/tools/mimirtool.md:162]: I would opt for singular tense, and fix all references.

**Review by osg-grafana — APPROVED** [review]
Unblocking with feedback

File: docs/sources/operators-guide/tools/mimirtool.md (hunk 9 above)
Comment (pracucci) [reply docs/sources/operators-guide/tools/mimirtool.md:162]: It let you load 1 or more rule groups, but I'm fine if you prefer to use singular tense. Going to change it.

File: docs/sources/operators-guide/tools/mimirtool.md (hunk 1 above)
Comment (pracucci) [reply docs/sources/operators-guide/tools/mimirtool.md]: Both ways to define an anchor are supported by [doc-validator](https://github.com/grafana/technical-documentation/tree/main/tools/doc-validator) (I've written the original one, then migrated to `grafana/technical-documentation` repo).

**Pull request merged**